sdk/python/foundation-models/system/inference/image-text-embeddings/image-text-embeddings-batch-endpoint.ipynb

```diff
@@ -0,0 +1,747 @@
+    "- /water_bottle\n",
+    "- /milk_bottle\n",
+    "- /carton\n",
+    "- /can\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import urllib\n",
+    "import shutil\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "# Change to a different location if you prefer\n",
+    "dataset_parent_dir = \"./data\"\n",
+    "\n",
+    "# create data folder if it doesnt exist.\n",
+    "os.makedirs(dataset_parent_dir, exist_ok=True)\n",
+    "\n",
+    "# Download data\n",
+    "download_url = \"https://cvbp-secondary.z19.web.core.windows.net/datasets/image_classification/fridgeObjects.zip\"\n",
+    "\n",
+    "# Extract current dataset name from dataset url\n",
+    "dataset_name = os.path.split(download_url)[-1].split(\".\")[0]\n",
+    "# Get dataset path for later use\n",
+    "dataset_dir = os.path.join(dataset_parent_dir, dataset_name)\n",
+    "\n",
+    "if os.path.exists(dataset_dir):\n",
+    "    shutil.rmtree(dataset_dir)\n",
+    "\n",
+    "# Get the data zip file path\n",
+    "data_file = os.path.join(dataset_parent_dir, f\"{dataset_name}.zip\")\n",
+    "\n",
+    "# Download the dataset\n",
+    "urllib.request.urlretrieve(download_url, filename=data_file)\n",
+    "\n",
+    "# Extract files\n",
+    "with ZipFile(data_file, \"r\") as zip:\n",
+    "    print(\"extracting files...\")\n",
+    "    zip.extractall(path=dataset_parent_dir)\n",
+    "    print(\"done\")\n",
+    "\n",
+    "# Delete zip file\n",
+    "os.remove(data_file)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.1 Arrange images in common folder for batch inference input"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dir_names = []\n",
+    "for dir_name in os.listdir(dataset_dir):\n",
+    "    dir_names.append(dir_name)\n",
+    "    dir_path = os.path.join(dataset_dir, dir_name)\n",
+    "    for path, subdirs, files in os.walk(dir_path):\n",
+    "        for file in files:\n",
+    "            shutil.move(os.path.join(path, file), dataset_dir)\n",
+    "\n",
+    "    shutil.rmtree(dir_path)\n",
+    "\n",
+    "labels = \", \".join(dir_names)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.2 Prepare a folder of CSV files with base64 images for batch inference input\n",
+    "\n",
+    "We can provide input images to batch inference in a csv file containing a column named \"image\" having either images in base64 format or publicly accessible image URLs. We provide empty string in the same csv file in the column named \"text\".\n",
```

Comment:
Nit: The new line at the end of this sentence is not being honored. Could you please check. The same issue is seen in other 3.* sections in this notebook